### ISSUE TITLE
Compatibility with Python 3.8

### ISSUE THREAD

**Issue by MPSystemsServices** [alonho/pql#29]
The Python ast module, which this library depends on has been changed in more recent versions of Python.  For sure 3.8, and possibly 3.7 or earlier.  The ast parser assigns a value of "Constant" to quoted strings and constant numeric values to the right of the comparison expression.  When this is handled by the resolve() method, the string "Constant" is appended to "handle_" and then the getattr() method fails because none of the field handler functions have a handle_Constant() function.  Hoping that this issue can be fixed.

**Comment by subbu-c** [comment]
You're right! I ran the example statement on python 3.8 (pql.find("a > 1 and b == 'foo' or not c.d == False")) and this is the error message I got:
![image](https://user-images.githubusercontent.com/70605075/97007788-ef650580-150f-11eb-9fdb-60df9af8d915.png)

**Comment by simplylu** [comment]
Same issue here, unfortunately.
